src/main/kotlin/lotto/LottoEnum.kt

```diff
@@ -0,0 +1,11 @@
+package lotto
+
+enum class LottoEnum(
+    val count: Int,
+    val prizeMoney: Int
+) {
+    THREE_COLLECT(3, 5000),
+    FOUR_COLLECT(4, 50000),
+    FIVE_COLLECT(5, 1500000),
+    SIX_COLLECT(6, 2000000000)
```

Comment:
```suggestion
    SIX_COLLECT(6, 2_000_000_000)
```
코틀린에서는 천 단위로 언더바를 추가하면 가독성이 좋아집니다.

Reply:
꿀팁 감사합니다!